scripts/repair_wheel.py

```diff
@@ -59,8 +59,8 @@ def main():
         additional_platforms = []
         if os_ == "macos":
             # first, get the target macOS deployment target from the wheel
-            match = re.match(r"^.*-macosx_(\d+)_(\d+)_x86_64\.whl$", file.name)
-            assert match is not None
+            match = re.match(r"^.*-macosx_(\d+)_(\d+)_.*\.whl$", file.name)
+            assert match is not None, f"Couldn't match on {file.name}"
```

Comment:
The wheels used to be misnamed (scikit-build didn't understand universal naming), and now they are correctly tagged, so this had to be modified to pick up the correct name.